lib/main.js

```diff
@@ -353,16 +352,17 @@ export default {
           }
 
           const msg = {
-            type: message.type,
-            filePath,
-            range,
+            severity: message.type.toLowerCase(),
```

Comment:
Indeed, it seems to only refer to ERROR and WARNING (always on all caps) on its messages. But, I can make sure there will be always only one of these three. If for some reason PHPCS return another type, should use 'warning' by default?